docs/usage/management-api-walkthrough/04_catalog.md

```diff
@@ -148,13 +167,12 @@ Consequently, if there may be more than one offer for the same Asset, requiring
 policies included.
 
 - The `@id` corresponds to the id of the Asset that can be negotiated for.
-- `dcat:Distribution` makes statements over which Data Planes an Asset's data can be retrieved. Currently, the EDC always
-  returns `HttpProxy` and `AmazonS3` even though not all EDC-deployments have both capabilities.
+- `dcat:Distribution` makes statements over which Data Planes an Asset's data can be retrieved. Currently, the TractusX-EDC supports
```

Comment:
Does this statement hold true only for the general capabilities of the Dataplane OR for also for this particular `dcat:Dataset`? With 0.6.x, I've witnessed it just returning `HttpData-PULL`, `AmazonS3-PUSH` and `AzureStorage-PUSH` independently of how the `Asset `was registered.

Reply:
That's independent on how the Asset is configured. The asset only says where the is resides and in which protocol, but that is the source of a transfer and the consumer does not see that. 

`HttpData-PULL`,  `AmazonS3-PUSH` and `AzureStorage-PUSH` are the destination, or how the consumer can receive data.

Those can be mixed, for example if I configure an asset with `HttpData` that can be consumed:

- HttpData-PULL (Consumer pull the data) 
- AmazonS3-PUSH/AzureStorage-PUSH (Provider push the data)